packages/client/components/NewAzureIssueInput.tsx

```diff
@@ -0,0 +1,249 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React, {FormEvent, useEffect, useRef, useState} from 'react'
+import {useFragment} from 'react-relay'
+import useAtmosphere from '~/hooks/useAtmosphere'
+import {MenuPosition} from '~/hooks/useCoords'
+import useMenu from '~/hooks/useMenu'
+import useMutationProps from '~/hooks/useMutationProps'
+import CreateTaskMutation from '~/mutations/CreateTaskMutation'
+import AzureDevOpsProjectId from '~/shared/gqlIds/AzureDevOpsProjectId'
+import {PALETTE} from '~/styles/paletteV3'
+import {NewAzureIssueInput_viewer$key} from '~/__generated__/NewAzureIssueInput_viewer.graphql'
+import useForm from '../hooks/useForm'
+import {PortalStatus} from '../hooks/usePortal'
+import UpdatePokerScopeMutation from '../mutations/UpdatePokerScopeMutation'
+import {CompletedHandler} from '../types/relayMutations'
+import convertToTaskContent from '../utils/draftjs/convertToTaskContent'
+import Legitity from '../validation/Legitity'
+import {CreateTaskMutationResponse} from '../__generated__/CreateTaskMutation.graphql'
+import Checkbox from './Checkbox'
+import Icon from './Icon'
+import NewAzureIssueMenu from './NewAzureIssueMenu'
+import PlainButton from './PlainButton/PlainButton'
+import StyledError from './StyledError'
+
+const StyledButton = styled(PlainButton)({
+  alignItems: 'center',
+  backgroundColor: 'transparent',
+  display: 'flex',
+  height: '20px',
+  justifyContent: 'flex-start',
+  margin: 0,
+  opacity: 1,
+  width: 'fit-content',
+  ':hover, :focus': {
+    backgroundColor: 'transparent'
+  }
+})
+
+const StyledIcon = styled(Icon)({
+  color: PALETTE.SKY_500,
+  fontSize: 20,
+  padding: 0,
+  alignContent: 'center'
+})
+
+const StyledLink = styled('a')({
+  color: PALETTE.SKY_500,
+  display: 'block',
+  fontSize: 12,
+  lineHeight: '20px',
+  textDecoration: 'none',
+  '&:hover,:focus': {
+    textDecoration: 'underline'
+  }
+})
+
+const Form = styled('form')({
+  display: 'flex',
+  flexDirection: 'column',
+  width: '100%'
+})
+
+const Item = styled('div')({
+  backgroundColor: PALETTE.SLATE_100,
+  cursor: 'pointer',
+  display: 'flex',
+  paddingLeft: 16,
+  paddingTop: 8,
+  paddingBottom: 8
+})
+
+const Issue = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  paddingLeft: 16,
+  width: '100%'
+})
+
+const NewIssueInput = styled('input')({
+  appearance: 'none',
+  background: 'transparent',
+  border: 'none',
+  color: PALETTE.SLATE_700,
+  fontSize: 16,
+  margin: 0,
+  padding: '0px 8px 0px 0px',
+  outline: 0,
+  width: '100%'
+})
+
+const Error = styled(StyledError)({
+  fontSize: 13,
+  textAlign: 'left',
+  width: '100%'
+})
+
+interface Props {
+  isEditing: boolean
+  meetingId: string
+  setIsEditing: (isEditing: boolean) => void
+  viewerRef: NewAzureIssueInput_viewer$key
+}
+
+const validateIssue = (issue: string) => {
+  return new Legitity(issue).trim().min(2, `C’mon, you call that an issue?`)
```

Comment:
+1 The error message feels not obvious :)

Reply:
That's a fair point. For me, I prefer using apps that feel playful, and I think informal language helps with that. It's also consistent with other parts of the app - we have 9 uses of the word "C'mon" 😄 

We could allow any length, but it's unlikely that the user meant to create an issue if they haven't hit the two-character requirement, so I think it's a better UX to give them that feedback